ThunderCloud/DateFormatter+Extensions.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  DateFormatter+Extensions.swift
+//  ThunderCloud
+//
+//  Created by Ben Shutt on 28/11/2019.
+//  Copyright © 2019 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+    
+    /// Standard `iso8601` `DateFormatter`
+    /// Note milliseconds including, for customization over `dateFormat`, see `iso8601(dateFormat:)`
+    static let iso8601: DateFormatter = {
```

Comment:
Can this call the `iso8601` function below, with `yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ` as format parameter?

Reply:
Like this suggestion @timxyz such Swift dev ;)

Reply:
Yeah makes a lot more sense! Oops